x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-resource-to-ecs@mappings.yaml

```diff
@@ -122,7 +136,16 @@ template:
       kubernetes.pod.uid:
         type: alias
         path: resource.attributes.k8s.pod.uid
+#      exception.type:
+#        type: alias
+#        path: attributes.exception.type
+#      exception.message:
+#        type: alias
+#        path: attributes.exception.message
```

Comment:
IIANM, the alias should be `error.exception.{type, message}` -> `exception.{type, message}`. And, we also need one for stacktrace: `error.stack_trace` -> `exception.stacktrace`